README.md

```diff
@@ -17,38 +17,37 @@ Features relating to the Local Editor are currently in the stage of alpha releas
 
 `corvid-cli` allows you to download your [Wix](https://www.wix.com) site and work on it locally. With this tool you can use your favorite development tools (e.g IDE, version control) to manage the code of your site, use the Wix Editor to edit the view, and collaborate with others in building your site. Read more about how to use the corvid-cli [here](https://support.wix.com/en/article/working-with-the-corvid-cli).
 
-## Getting Started
-m
-### Prerequisites
-Before doing anything else, you need to make sure you have [Node.JS](https://nodejs.org)  version >= 10.0.0 installed and working.
 
+## Setup
+
+### Prerequisites
+[Node.JS](https://nodejs.org) version >= 10 
 
 ### Getting ready to use corvid-cli
 
-Create a new empty directory that will contain your site files and cd into it:
-```
-mkdir <any-folder-name>
-cd <any-folder-name>
-```
+Just run:
 
-Initialize a package.json file so that you can install local dependencies:
 ```
-npm init -y
+npx create-corvid-app <folder-name> [your-wix-site-url]
 ```
 
-Install corvid-cli as a local development dependency:
+Sit back and relax while we initialize the specified folder with the recommended setup for working with `corvid-cli`.
+If you provide a Wix site URL (optional), we will also download your site into that folder.
+
+
+## Work on your site
+
+Make sure you get into the folder you've setup:
 ```
-npm install --save-dev corvid-cli corvid-types
+cd <folder-name>
 ```
 
-### Work on your site
-
-Download your exising wix site locally:
+If you didn't provide your Wix site URL during setup, download your exising wix site locally by running:
```

Comment:
If you didn't...URL when running `create-corvid-app`, download your site by running: